packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -183,6 +195,8 @@ function getProfileFromEvent(event: Event) {
     brandColor: profile.brandColor,
     darkBrandColor: profile.darkBrandColor,
     theme: profile.theme,
+    bookerLayouts:
+      event.bookerLayouts || ("defaultBookerLayouts" in profile ? profile.defaultBookerLayouts : null),
```

Comment:
If the event has a layout set, we pass that. If there's no layout set, we default to the profile settings of the user. If that's not set, we use null, which in the booker will be resolved to showing all 3 layouts with the month_view as default.